### DISCUSSION TITLE
Which Libary for UR10e

### DISCUSSION THREAD

**Discussion by SebastianKoehler1** [visose/Robots#177] [Q&A] [answered]
Hi all,
which library would you prefer for the Universal Robot UR10e? I see the UR10 at Bartlett, KULeuven, AAU...
Thanks in advance

**Comment by visose** [comment, marked as answer]
Hi @SebastianKoehler1, you can check in the [libraries](https://github.com/visose/Robots/tree/libraries) branch if there are any differences between them. You don't need to use the available libraries, you can create your own xml/3dm pair of files using the existing ones as base.
